### ISSUE TITLE
High-level commentary on WCAG 2.1

### ISSUE THREAD

**Issue by alexswli** [w3c/wcag21#205]
I submitted a series of comments at the success criterion level today on behalf of Microsoft.  But I wish to abstract my observations and hope this will help the working group in propelling WCAG forward.

One reason for WCAG 2.0’s success is its board applicability across all types of web content.  The working group took great effort in making sure that WCAG is universally applicable.  It is true that certain types of content take more work than others to meet WCAG 2.0 AA level.  But we made it feasible for all content to meet the SC as long as they put in the effort.  When certain content types have essential reasons to not follow an SC in full, we carved out exceptions appropriately or moved the SC to AAA.  This is one reason why WCAG 2.0 is a successful standard today.  It occurs to me that some of proposed SC are taking an approach of forbidding certain classes of content instead of finding ways to include them.  If WCAG 2.1 becomes a standard that only certain types of content can meet, it will no longer be suitable for the entire market and no longer be relevant for regulatory purposes.  The working group should consider this implication seriously.

Another observation I have is that some of the proposed SC seem to be putting all the responsibilities of accessibility on the shoulders of content authors.  There is reasonable work for browsers, OS, AT, and hardware to do to make user experience accessible.  Some responsibilities even rest on the consumers of the content.  Content author should be asked to do their share instead of fixing problems that’s better solved by other entities.  As a representative of a company that makes OS, browser, authoring tools, hardware, and arguably some AT, it strikes me that we can, in some cases, improve accessibility more consistently and effectively without asking content authors to know everything about how AT, OS, and browsers function and to work around all the ever-shifting problems.  WCAG 2.0 was successful, in part, due to the recognition of this separation of responsibilities.

Lastly, WCAG 2.0 makes every SC objectively testable.  This is a critical feature that makes WCAG 2.0 suitable for cross market application.  Web content is designed to be democratized and to keep up in real time with the world.  It is not realistic to expect content authors to convene a committee and to gain consensus before publication of content.  If WCAG 2.1 is to retain WCAG 2.0's real world relevancy, we must not forget this important aspect.

I want to congratulate the working group for the new draft and look forward to the debates ahead.  There are many good ideas in this document—some belong in WCAG, some probably belong elsewhere.  In any case, I am confident that we can make timely and pragmatic forward progress in accessibility based on many of these ideas.

**Comment by awkawk** [comment]
@alexswli thanks for the wise words. I'm closing this as an issue, but the points you raise are important to many on the group so I'm confident that we will find a path forward.
